.github/workflows/tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+concurrency:
+  group: ${{ github.repository }}-${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  tests_linux:
```

Comment:
This section is actually redundant with the tests in `makefile.yml`.

Reply:
good catch, i'm hoping to move away from the makefile.yml as this test.yml comes together